tests/test_primitives/test_point/test_point_serializers.py

```diff
@@ -67,15 +67,15 @@ def test_bytes_serializers(point_bytes, nid, curve):
     for point_representation in representations:
 
         malformed_point_bytes = point_representation + b'0x'
-        with pytest.raises(ValueError):
+        with pytest.raises(InternalError):
```

Comment:
....as opposed to an external one?

Reply:
`ValueError` was raised by our serialization logic, while `InternalError` is what cryptography.io raises to account for OpenSSL errors.